test/create-probot.test.ts

```diff
@@ -83,7 +83,7 @@ describe("createProbot", () => {
     expect(probot.log.level).toEqual("trace");
   });
 
-  test("defaults, custom host", () => {
+  test.skip("defaults, custom host", () => {
```

Comment:
Can this test be made to work at all?
or is it better just being deleted?

Reply:
let me investigate

Reply:
@wolfy1339 

Please check my commit :)